torch/_functorch/_aot_autograd/subclass_utils.py

```diff
@@ -165,16 +258,28 @@ def wrap_tensor_subclasses(
 ) -> Tuple[Any, ...]:
     wrapped_args = []
     num_args_tallied = 0
+    num_args = sum(
+        1 if isinstance(meta, int) else meta.arg_count for meta in subclass_metas
+    )
+    count_extra = len(unwrapped_args) > num_args
```

Comment:
fyi @mlazos (tldr here is that for any graph inputs from dynamo that are subclasses that have dynamic shapes, we now add their SymInt sizes as extra graph args, which makes this remapping a bit more complicated)